src/eval/datetime.rs

```diff
@@ -37,60 +45,102 @@ impl Datetime {
         formatted_result.map_err(format_time_format_error)
     }
 
-    pub fn date(&self) -> Option<time::Date> {
+    /// Return the year of the datetime, if existing.
+    pub fn year(&self) -> Option<i32> {
         match self {
-            Datetime::Date(date) => Some(*date),
+            Datetime::Date(date) => Some(date.year()),
             Datetime::Time(_) => None,
-            Datetime::Datetime(datetime) => Some(datetime.date()),
+            Datetime::Datetime(datetime) => Some(datetime.year()),
         }
     }
 
-    pub fn time(&self) -> Option<time::Time> {
+    /// Return the month of the datetime, if existing.
+    pub fn month(&self) -> Option<u8> {
+        match self {
+            Datetime::Date(date) => Some(date.month().into()),
+            Datetime::Time(_) => None,
+            Datetime::Datetime(datetime) => Some(datetime.month().into()),
+        }
+    }
+
+    /// Return the weekday of the datetime, if existing.
```

Comment:
Nitpick: Maybe document that Monday=0.

Reply:
Monday should actually be 1. 🤔 Did you try it yourself?

Reply:
No, did not try.

Reply:
Okay, I think it should work though, as I also added one test for this. But thanks! :D